deployments/helm/README.md

```diff
@@ -31,14 +31,15 @@ eBPF-based autoinstrumentation of HTTP and HTTPS services
 | namespaceOverride | string | `""` | Override the deployment namespace |
 | nodeSelector | object | `{}` | The nodeSelector field allows user to constrain which nodes your DaemonSet pods are scheduled to based on labels on the node |
 | podSecurityContext | object | `{}` |  |
+| preset | string | `"application"` | Preconfigures some default properties for network or application observability. Accepted values are "network" or "application". |
```

Comment:
Is there a possibility to deploy both at the same time?

Reply:
Yes, you can provide te configuration YAML as for any other Beyla configuration. The "preset" is only a shortcut. This could be an array but then the `--set preset=["network"]` syntax looked more cumbersome.